test/AutoRemediateS3-002.test.js

```diff
@@ -178,37 +152,40 @@ describe('S3-002 AutoRemediation', () => {
           Owner: expect.any(Object)
         }
       }
-      expect(putBucketAclMock).toHaveBeenCalledWith(expect.objectContaining(expectedGrant), expect.any(Function))
+      expect(mockS3).toHaveReceivedCommandWith(PutBucketAclCommand, expectedGrant)
     })
   })
 
   describe('invalid invocation', () => {
-    const mockCallback = (done) => {
-      return (err, data) => {
-        expect(err).toBeDefined()
-        expect(getBucketAclMock).not.toHaveBeenCalled()
-        expect(putBucketAclMock).not.toHaveBeenCalled()
-        done()
-      }
-    }
-
-    it('should fail when event is undefined', done => {
-      source.handler(undefined, jest.fn(), mockCallback(done))
+    it('should fail when event is undefined', async () => {
+      await expect(source.handler(undefined, jest.fn()))
+        .rejects
+        .toThrow('Invalid event')
+      expect(mockS3).not.toHaveReceivedCommand(GetBucketAclCommand)
+      expect(mockS3).not.toHaveReceivedCommand(PutBucketAclCommand)
     })
 
-    it('should fail when "resource" missing from the event', done => {
+    it('should fail when "resource" missing from the event', async () => {
       const malformedEvent = {
         ruleId: 'S3-002'
       }
-      source.handler(malformedEvent, jest.fn(), mockCallback(done))
+      await expect(source.handler(malformedEvent, jest.fn()))
```

Comment:
♻️ no second argument needed
```suggestion
      await expect(source.handler(malformedEvent))
```